src/control/actor/host/notify.go

```diff
@@ -59,39 +58,32 @@ func (c *HostNotifyCmd) listenCloseF(net network.Network, addr ma.Multiaddr) {
 
 func (c *HostNotifyCmd) connectedF(net network.Network, conn network.Conn) {
 	logrus.WithFields(logrus.Fields{
-		"EVENT": "Connection detected",
-		"DIRECTION": fmtDirection(conn.Stat().Direction),
+		"EVENT":     "Connection detected",
+		"DIRECTION": conn.Stat().Direction.String(),
 	}).Info("Peer: ", conn.RemotePeer().String())
-
 	// try to request metadata for the peer
 	peerData, err := PollPeerMetadata(conn.RemotePeer(), c.Base, c.PeerMetadataState, c.Store, c.PeerStore)
-
 	// double check that the peerData is not empty
-	if err == nil && peerData.String() != "no data available" {
-		// create the peer
-		peer := metrics.NewPeer(conn.RemotePeer().String())
-		peer, err = fetchPeerExtraInfo(peerData)
-		if err != nil {
-			log.Error("Could not fetch peer data for: " + conn.RemotePeer().String())
-		}
-		logrus.WithFields(logrus.Fields{
-			"EVENT": "Metadata request OK",
-		}).Info("Peer: ", conn.RemotePeer().String())
-		
-		// store the peer and record that the connection was ok
-		c.PeerStore.StoreOrUpdatePeer(peer)
-	} else {
+	if err != nil {
 		logrus.WithFields(logrus.Fields{
 			"EVENT": "Metadata request NOK",
```

Comment:
Don't you think that printing an error here is too much? I mean, its kind of expected to get metadata request fails for some peers right? I would just warn